ui/src/components/SecretForm/index.jsx

```diff
@@ -226,6 +224,7 @@ export default class SecretForm extends Component {
             {showSecret && (
               <ListItem className={classes.editorListItem}>
                 <CodeEditor
+                  placeholder="foo:bar"
```

Comment:
Agreed with @djmitche, let's do `placeholder="YAML representation of secret data"`,

Reply:
okay. let me do that.